components/Favourites/Favourites.js

```diff
@@ -0,0 +1,144 @@
+import PropTypes from "prop-types";
+
+import { Divider, ItemCard } from "../../styles/UIKit";
+import { FAVOURITES_TITLE } from "../../utils/constants";
+
+const responsive = {
+  superLargeDesktop: {
+    // the naming can be any, depends on you.
+    breakpoint: { max: 4000, min: 3000 },
+    items: 5,
+  },
+  desktop: {
+    breakpoint: { max: 3000, min: 1024 },
+    items: 5.5,
+  },
+  tablet: {
+    breakpoint: { max: 1024, min: 464 },
+    items: 2,
+  },
+  mobile: {
+    breakpoint: { max: 464, min: 0 },
+    items: 1,
+  },
+};
+
+let NextOne, PreviousOne;
+
+const ButtonGroup = ({ next, previous, ..._ }) => {
+  NextOne = next;
+  PreviousOne = previous;
+  return <div></div>;
+};
+
+const Favourites = () => {
+  const GotoNext = () => {
+    NextOne();
+  };
+  const GotoPrevious = () => {
+    PreviousOne();
+  };
+
+  return (
+    <FavoritesContainer>
+      <FavouritesTitleContainer>
+        <FavouritesTitle>{FAVOURITES_TITLE}</FavouritesTitle>
+        <FavouriteArrowContainer>
+          <FavouriteArrow onClick={GotoPrevious}> &lt; </FavouriteArrow>
+          <FavouriteArrow onClick={GotoNext}> &gt;</FavouriteArrow>
+        </FavouriteArrowContainer>
+      </FavouritesTitleContainer>
+
+      <Divider />
+      <CarouselContainer>
+        <Carousel
+          arrows={false}
+          responsive={responsive}
+          customButtonGroup={<ButtonGroup />}
+          renderButtonGroupOutside={true}
+        >
+          <ItemCard
+            itemName={"Greta White Midi Dress"}
+            itemPrice={"159.00"}
+          ></ItemCard>
+          <ItemCard
+            itemName={"Lilika Silver Mini"}
+            itemPrice={"99.00"}
+          ></ItemCard>
+          <ItemCard
+            itemName={"Crystal Off White Dress"}
+            itemPrice={"250.00"}
+          ></ItemCard>
+          <ItemCard
+            itemName={"Carson Shoulder"}
+            itemPrice={"175.00"}
+          ></ItemCard>
+          <ItemCard
+            itemName={"Dylan Polo Shirt"}
+            itemPrice={"250.00"}
+          ></ItemCard>
+          <ItemCard
+            itemName={"Greta White Midi Dress"}
+            itemPrice={"159.00"}
+          ></ItemCard>
+          <ItemCard
+            itemName={"Lilika Silver Mini"}
+            itemPrice={"99.00"}
+          ></ItemCard>
+          <ItemCard
+            itemName={"Crystal Off White Dress"}
+            itemPrice={"250.00"}
+          ></ItemCard>
+          <ItemCard
+            itemName={"Alexander Immanuel"}
+            itemPrice={"175.00"}
+          ></ItemCard>
+          <ItemCard
+            itemName={"Dylan Polo Shirt"}
+            itemPrice={"250.00"}
+          ></ItemCard>
+        </Carousel>
+      </CarouselContainer>
+    </FavoritesContainer>
+  );
+};
+
+export default Favourites;
+
+ButtonGroup.propTypes = {
+  next: PropTypes.func,
+  previous: PropTypes.func,
+};
+
+const FavouritesTitleContainer = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
FYI flex-direction:row is a default property when you apply display: flex to any element, so even if you remove it this will be rendered in a row only.